client/src/pages/_app.tsx

```diff
@@ -1,3 +1,4 @@
+import 'reflect-metadata';
```

Comment:
Yes, I think because I added `babel-plugin-transform-typescript-metadata` previously I was getting errors related to reflect.metadata and had to add this. Removed the package.